packages/terra-core-docs/src/terra-dev-site/doc/form-select/example/combobox/AllowClear.jsx

```diff
@@ -6,7 +6,7 @@ import styles from '../FormSelectDocCommon.module.scss';
 const cx = classNames.bind(styles);
 
 const AllowClearExample = () => (
-  <Combobox placeholder="Select a color" allowClear className={cx('form-select')}>
+  <Combobox placeholder="Select a color" ariaLabel="Select a color" allowClear className={cx('form-select')}>
```

Comment:
instead of adding aria-label on each of the example can you make similar changes for combobox to auto assign placeholder text to visuallyHiddenText or Aria-Label as made on this PR : https://github.com/cerner/terra-core/pull/3984

Reply:
Updated - https://github.com/cerner/terra-core/pull/3976/commits/4a68901c81bda93178cea45b9902d18c7346a166